### PR TITLE
lpae: disable CONFIG_QCOM_PIL_INFO to fix build on armv7

### PR DESCRIPTION
**Pull request by ggardet**

### CODE REVIEW

File: config/armv7hl/lpae [hunk 1]
```diff
@@ -869,7 +869,7 @@ CONFIG_CRYPTO_CHACHA20_NEON=m
 CONFIG_CRYPTO_POLY1305_ARM=m
 CONFIG_CRYPTO_NHPOLY1305_NEON=m
 CONFIG_CRYPTO_CURVE25519_NEON=m
-CONFIG_AS_VFP_VMRS_FPINST=y
+# CONFIG_AS_VFP_VMRS_FPINST is not set
```
Comment (dirkmueller) [inline config/armv7hl/lpae:872]: this change is unrelated - do you have a too old gas / gcc installed?

**Comment by dirkmueller** [comment]
I've submitted this as users/dmueller/stable/for-next

File: config/armv7hl/lpae (hunk 1 above)
Comment (ggardet) [reply config/armv7hl/lpae:872]: I did the update on my laptop, so not an armv7 gcc. That could explain it.

**Comment by ggardet** [comment]
@dirkmueller I guess we could close this PR as it has been submitted internally?
